addons/website_slides/static/src/js/public/components/slide_quiz_finish_dialog/slide_quiz_finish_dialog.js

```diff
@@ -0,0 +1,147 @@
+/** @odoo-module **/
+
+import { Component, onMounted, useState } from "@odoo/owl";
+import { browser } from "@web/core/browser/browser";
+import { Dialog } from "@web/core/dialog/dialog";
+import { _t } from "@web/core/l10n/translation";
+
+export class SlideXPProgressBar extends Component {
```

Comment:
I think we should put this component in separate files (fine in the same directory for now).